src/bosh_aws_cpi/lib/cloud/aws/instance_manager.rb

```diff
@@ -86,6 +90,29 @@ def babysit_instance_creation(instance, vm_cloud_props)
       end
     end
 
+    def attach_ip_prefixes(instance, private_ip_addresses)
+      private_ip_addresses.each do |address|
+          private_ip = address[:ip]
+          prefix = address[:prefix].to_s
+        
+          if @param_mapper.ipv6_address?(private_ip)
+            if !prefix.empty? && prefix.to_i < 128
+              @ec2.client.assign_ipv_6_addresses(
+                network_interface_id: instance.network_interface_id,
+                ipv_6_prefixes: ["#{private_ip}/#{prefix}"] # aws only supports /80 prefixes
```

Comment:
if `/80` is the only valid length, should this check be stricter? Same for the ipv4 check below. 

(assuming this follows https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-prefix-eni.html the only valid values are `/80` and `/28` so `<128` and `<32` seem like they would allow invalid prefixes; is the intent to delegate validation to AWS and just bubble up the error?)

Reply:
yes, the intent is to delegate the validation to AWS, since in the future they might support other prefixes, and we thought it would be nice to already be compatible